src/os/bluestore/HMSMRDevice.cc

```diff
@@ -0,0 +1,1202 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2014 Red Hat
+ *
```

Comment:
might want to update the date. it should be the first year (date) the creative work is published.

also @agayev, do you intend to transfer the copyright of your code to Red Hat? if this is a work made for hire, i.e. this work is created because your employer hires you to do so. normally, the copyright belongs to your employer.

Reply:
Done.  For now, I've kept the copyright for myself.  I will update it if needed.